store/postgres/src/store.rs

```diff
@@ -459,6 +462,54 @@ impl Store {
         Ok(())
     }
 
+    /// Build a partial Postgres index on a Subgraph-Entity-Attribute
+    fn build_entity_attribute_index_with_conn(
+        &self,
+        conn: &PgConnection,
+        operation: AttributeIndexOperation,
+    ) -> Result<(), Error> {
+        let (index_type, index_operator) = match operation.field_value_type {
+            ValueType::Boolean
+            | ValueType::BigInt
+            | ValueType::Bytes
+            | ValueType::Float
+            | ValueType::ID
+            | ValueType::Int => (String::from("btree"), String::from("")),
+            ValueType::String => (String::from("gin"), String::from("gin_trgm_ops")),
+        };
+
+        select(build_attribute_index(
+            operation.subgraph_id.to_string(),
+            operation.index_name.clone(),
+            index_type,
+            index_operator,
+            operation.attribute_name.clone(),
+            operation.entity_name.clone(),
+        ))
+        .execute(conn)
+        .map_err(Error::from)
+        .and_then(move |row_count| match row_count {
+            1 => Ok(()),
+            _ => Err(format_err!(
+                "failed to create indexes for subgraph:{:?}",
```

Comment:
Our convention is for non-panic errors to start with a capital letter. Also, there's a space missing after the `subgraph:`. I'd make it `{}` for the subgraph ID formatting too, if possible.